nexus/swags/models.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Third Party Stuff
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+# nexus Stuff
+from nexus.base.models import ImageMixin, TimeStampedUUIDModel
+
+
+class Swag(ImageMixin, TimeStampedUUIDModel):
+    item = models.ForeignKey('SwagItem', on_delete=models.CASCADE, verbose_name=_('Item'), null=False, blank=False)
+    users = models.ManyToManyField('users.User', through='SwagOwnership')
+    description = models.TextField(verbose_name=_('Description'), null=False, blank=False)
+
+    class Meta:
+        verbose_name = _('Swag')
+        verbose_name_plural = _('Swags')
+        db_table = 'swags'
+
+    def __str__(self):
+        return self.item.name
```

Comment:
This will create duplicity because there can be *many* `Swags` corresponding to *one* `SwagItem`. This `__str__` will return the same name for all such `Swags`.

Reply:
So, what else can we return here?

Reply:
Probably also the item that it corresponds to. It depends on how you want to use them.